sdks/python/apache_beam/runners/google_cloud_dataflow/internal/clients/dataflow/__init__.py

```diff
@@ -20,8 +20,8 @@
 
 import pkgutil
 
+from apitools.base.py import *
```

Comment:
Yeah lets remove it if we don't need it as we shouldn't be exposing the underlying library if we don't need it.